acceptance/config_test.go

```diff
@@ -31,6 +37,10 @@ type TestConfig struct {
 	// If true, run this test when running with cloud env configured
 	Cloud *bool
 
+	// Which stage features this test requires. Each string is a feature name that must be present
+	// in the current stage for the test to run. If absent, no specific features are required.
+	RequiredStageFeatures []StageFeature
```

Comment:
Let's keep this simple and composable and use *bool per feature.

RequiresUnityCatalog *bool

Reply:
fixed in a follow-up commit